apps/sms/views/shared/js/activity_handler.js

```diff
@@ -452,12 +461,13 @@ var ActivityHandler = {
             contacts[0].name.length && contacts[0].name[0]) {
             sender = contacts[0].name[0];
           }
+
           if (message.type === 'sms') {
-            continueWithNotification(sender, message.body || '');
+            return continueWithNotification(sender, message.body || '');
           } else { // mms
```

Comment:
no need in ```else``` since we return. Or maybe even like this (up to you)?:
```
var titlePromise = message.type === 'sms' ? Promise.resolve(message.body || '') : getTitleForMms();
return titlePromise.then((title) => {
  return continueWithNotification(sender, title);
});
```